uwblueprint/jump-math
# chore: refactor chakra ui component theming

## Notion ticket link

N/A



## Implementation description
* Added `alert.ts`, `form.ts`, `formError.ts`, and `input.ts` to provide default theming for the corresponding Chakra UI components based on designs
* Deleted `EmailInput.tsx`, `TextInput.tsx`, `InputLabel.tsx`, and `TextInput.tsx`, since they became redundant with the introduction of the above themes
* `AddAdminModal.tsx`: replaced the instances of `EmailInput`, `TextInput`, `InputLabel`, and `TextInput` with the original Chakra UI components (and fixed the email input padding)



## Steps to test
1. Make sure the appearances of inputs and alerts across all forms match designs (this mostly involves the add admin modal)



## What should reviewers focus on?


## Checklist
